### PR TITLE
[BPK-2333] Add theme to chip

### PR DESCRIPTION
**Pull request by tiagoengel**
Screenshot_1554455947
Screenshot_1554455971

+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/master/CONTRIBUTING.md)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

**Comment by georgegillams** [comment]
Thank you for your contribution.
You can use the following dependency to test your application while we review your pull request:

`com.github.Skyscanner:backpack-android:b10e55b5318efb59acb26cbdbb5a94407bfc8c2c`

File: app/src/main/res/values/themes.xml [hunk 1]
```diff
@@ -73,7 +82,8 @@
   <style name="DohaThemeButtonPrimary">
     <item name="buttonBackgroundColor">#ffb300</item>
     <item name="buttonTextColor">@color/bpkWhite</item>
-  </style>
+  </style
```
Comment (amolgupta) [inline app/src/main/res/values/themes.xml]: does the lint plugin not catch this?

File: app/src/main/res/values/themes.xml [hunk 2]
```diff
@@ -36,15 +37,22 @@
     <item name="buttonStrokeColor">#5E072C</item>
   </style>
 
+  <style name="LondonThemeChip">
+    <item name="chipSelectedBackgroundColor">?colorPrimary</item>
```
Comment (amolgupta) [inline app/src/main/res/values/themes.xml]: we can call it only `chipColor` as there are no other attributes

File: app/src/main/res/values/themes.xml (hunk 1 above)
Comment (tiagoengel) [reply app/src/main/res/values/themes.xml]: AFAIL it doesn't lint xml

File: app/src/main/res/values/themes.xml (hunk 2 above)
Comment (tiagoengel) [reply app/src/main/res/values/themes.xml]: We follow this pattern everywhere else so it's better we keep it consistent, see: https://github.com/Skyscanner/backpack/blob/master/decisions/theme-props-naming.md

**Pull request merged**